readme.md

```diff
@@ -25,6 +27,16 @@ go run .
 # All done, the bot should be running now.
 ```
 
+# Setup with Docker
```

Comment:
this section should probably be above `Automatic setup with Nix`, as Docker is more popular